PyBERT.sublime-project

```diff
@@ -2,7 +2,13 @@
 	"folders":
 	[
 		{
-			"path": "."
-		}
+			"path": ".",
+		},
+		{
+			"path": "/Users/dbanas/Documents/gitit",
```

Comment:
There are your local personal paths.  Doesn't bother me having it in (vscode user myself) but if someone tried to open pybert with sublime; it'd fail for them.

Reply:
Adding this file was a mistake.
I'll remove it.